src/usecases/configure-figma-teams-use-case.ts

```diff
@@ -0,0 +1,29 @@
+import {
+	type ConnectInstallation,
+	FigmaTeamAuthStatus,
+} from '../domain/entities';
+import { figmaService } from '../infrastructure/figma';
+import { figmaTeamRepository } from '../infrastructure/repositories';
+
+export const configureFigmaTeamUseCase = {
+	execute: async (
+		teamId: string,
+		atlassianUserId: string,
+		connectInstallation: ConnectInstallation,
+	): Promise<void> => {
+		const { webhookId, teamId: figmaTeamId } =
+			await figmaService.createFileUpdateWebhook(
+				teamId,
+				atlassianUserId,
+				connectInstallation.sharedSecret,
+			);
+		await figmaTeamRepository.upsert({
+			webhookId,
+			teamId: figmaTeamId,
+			teamName: 'TODO',
```

Comment:
We're still deciding how to handle team names, there's an ongoing LDR. This column may actually not be required, so I'm just leaving this as 'TODO' for now.